stbt_rig.py

```diff
@@ -1249,7 +1249,11 @@ def pytest_sessionstart(session):
         session.stbt_args = args
         resolve_args(session.stbt_args)
 
-        capmanager = session.config.pluginmanager.getplugin('capturemanager')
+        pluginmanager = session.config.pluginmanager
+        if not session.config.option.collectonly:
+            pluginmanager.unregister(name="python")
```

Comment:
does this mean that we can remove `'-p', 'no:python',` from our tests? (`test_stbt_rig.py`)

Reply:
I suppose we could. It was nice to see that `pluginmanager.unregister` doesn't raise if it was already disabled from the command line, as that's what our current PyCharm documentation recommends.